apps/src/gamelab/AnimationTab/AnimationTab.jsx

```diff
@@ -13,6 +15,13 @@ import AnimationList from './AnimationList';
 import FrameList from './FrameList';
 import ResizablePanes from './ResizablePanes';
 
+/**
+ * @const {string} domain-relative URL to Piskel index.html
+ * In special environment builds, append ?debug flag to get Piskel to load its own debug mode.
+ */
+const PISKEL_PATH = '/blockly/js/piskel/index.html' +
+    (PISKEL_DEVELOPMENT_MODE ? '?debug' : '');
```

Comment:
A second approach here would be to make PISKEL_PATH the global provided by the build script.  Better/worse?